src/content/en/fundamentals/engage-and-retain/push-notifications/permissions-subscriptions.markdown

```diff
@@ -180,3 +183,34 @@ if ('showNotification' in ServiceWorkerRegistration.prototype) {
   });
 }
 {% endhighlight %}
+
+### Upgrading from GCM to the web push protocal {#upgrad}
```

Comment:
I'd drop this section completely (along with any mention of the manifest) and move this into a "Legacy Browser Support" or "Non-Spec Compliant Browsers". Some that suggest it's not the normal flow / path.